pkg/splunk/alert.go

```diff
@@ -35,7 +38,10 @@ func (w Webhook) Details() AlertDetails {
 func (w Alert) Details() []AlertDetails {
 	alerts := []AlertDetails{}
 	for _, result := range w.SearchResults.Results {
-		alerts = append(alerts, NewAlertDetails(result))
+		alert := NewAlertDetails(result)
+		if alert.Valid() {
```

Comment:
What should happen here if alert details are not  valid?  Should an error be returned?  Maybe I can handle this case as part of or related to OSD-14711